src/screens/ChannelListScreenV2/IncognitoEmptyChat/index.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import {Image, Text, View, StyleSheet, TouchableOpacity, Dimensions} from 'react-native';
+import {useNavigation} from '@react-navigation/core';
+
+import EmptyIncognito from '../../../assets/images/empty_incognito.png';
+import IconArrowRight from '../../../assets/icons/Ic_arrow_right';
+import dimen from '../../../utils/dimen';
+import {COLORS} from '../../../utils/theme';
+import {fonts, normalizeFontSize} from '../../../utils/fonts';
+import {ANONYMOUS} from '../../../hooks/core/constant';
+import useAnonymousChannelListScreenHook from '../../../hooks/screen/useAnonymousChannelListHook';
+import {DISCOVERY_TAB_TOPICS, NavigationConstants} from '../../../utils/constants';
+
+const {width: widthScreen} = Dimensions.get('window');
+
+const MODE_FULL = 1;
+const MODE_BUTTON = 2;
+const MODE_HIDE = 3;
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The constants `MODE_FULL`, `MODE_BUTTON`, and `MODE_HIDE` are used to represent different states of the component. It would be more maintainable and readable if these were encapsulated in an object or TypeScript enum.

```diff
- const MODE_FULL = 1;
- const MODE_BUTTON = 2;
- const MODE_HIDE = 3;
+ const DisplayMode = {
+   FULL: 1,
+   BUTTON: 2,
+   HIDE: 3,
+ };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->